.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Slimefun
+This document outlines various ways how you can help contribute to Slimefun and make this a bigger and better project.<br>
```

Comment:
I feel like "help contribute to" part has some grammatical errors. I'd've used "help by contributing to" and "and make this " to "and making this"

Reply:
Help + infitive without the to is gramatically correct and the informal version of it.